PackagePrice.java

```diff
@@ -0,0 +1,52 @@
+package pricecomparison;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+public class PackagePrice {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		WebDriver driver = new FirefoxDriver();
+
+		String appUrl = "http://www.tripfactory.com/package/get-price-req?pkgId=27433&ahdate=06%2F05%2F2016";
+		driver.get(appUrl);
+
+		WebElement wE = driver.findElement(By.xpath(".//*[@id='invCalMth2']/table"));
+
+		System.out.println(wE.getText());
+
+		driver.findElement(By.xpath(".//*[@id='invCalMth2']/table/tbody/tr[2]/td[1]/div[2]/div[3]")).click();
+		System.out.println("selecting date: 1/05/2016");
+		driver.findElement(By.xpath(".//*[@id='calculateButton']/a")).click();

```

Comment:
Some developers repeat same ids on a page. THat should be caught as an error and reported.
SO your script should do such basic checks. If one day someone adds an additional element with 'calculateButton', your script should throw an exception (https://docs.oracle.com/javase/tutorial/essential/exceptions/index.html) for duplicate ids. 

As an automation tester, you need to assure sanctity of every element you are accessing. And if you are using IDs, and there are unresolved conflicts, your code can potentially break.